apps/settings/js/panels/call_fdn_settings/panel.js

```diff
@@ -0,0 +1,103 @@
+define(function(require) {
+  'use strict';
+
+  var SettingsPanel = require('modules/settings_panel');
+  var DialogService = require('modules/dialog_service');
+
+  return function ctor_call_fdn_settings_panel() {
+    return SettingsPanel({
+      onInit: function(panel, options) {
+        this._cardIndex = options.cardIndex || 0;
```

Comment:
When back from the fdn list, there won't be `cardIndex` but we still need to use the same card index, so we need to cache the index.